src/common/options.rs

```diff
@@ -145,19 +145,62 @@ impl Endpointing {
     }
 }
 
+/// Endpointing value
+///
+/// See the [Deepgram Endpointing feature docs][docs] for more info.
+///
+/// [docs]: https://developers.deepgram.com/docs/endpointing
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Copy)]
+pub enum Summarize {
+    #[allow(missing_docs)]
+    Enabled(bool),
+
+    #[allow(missing_docs)]
+    V2,
+}
+
+/// Endpointing impl
+impl Summarize {
+    #[allow(missing_docs)]
+    pub fn as_str(&self) -> String {
+        match self {
+            Summarize::Enabled(value) => {
+                if *value {
+                    "true".to_string()
+                } else {
+                    "false".to_string()
+                }
+            },
```

Comment:
```suggestion
            Summarize::Enabled(value) => value.to_string(),
```

Same here, just call `to_string` on the boolean directly.